config/sync/search_api_autocomplete.search.contacts.yml

```diff
@@ -1,6 +1,6 @@
 uuid: e9a68046-b084-4ffe-8570-4f9a9c1a606c
 langcode: en
-status: true
+status: false
```

Comment:
Mirrors NIDirect D7 site behaviour. Can always revert this chunk if we need autocomplete in future on contacts search.